ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml

```diff
@@ -2,8 +2,7 @@
 chain:
   as: openshift-qe-control-plane
   steps:
-  - chain: openshift-qe-control-plane-smoke
```

Comment:
Any reason, why we are removing smoke-test?

Reply:
Removing smoke test because I implemented the warmup runs in each workload itself.